src/jobs_auto_report/utils.py

```diff
@@ -0,0 +1,46 @@
+from models import JobDetails, JobEquinixDetails, JobRefs
+
+class Utils:
+
+    @staticmethod
```

Comment:
instead of grouping those methods inside the class ``Utils`` (which is pretty generic as  a name)
just put those under the scope of the ``utils.py`` module